src/main/kotlin/org/ptss/support/security/JwtValidator.kt

```diff
@@ -0,0 +1,32 @@
+package org.ptss.support.security
+
+import io.quarkus.security.UnauthorizedException
+import io.smallrye.jwt.auth.principal.JWTParser
+import jakarta.enterprise.context.ApplicationScoped
+import jakarta.inject.Inject
+import org.ptss.support.domain.enums.Role
+
+@ApplicationScoped
+class JwtValidator @Inject constructor(
+    private val jwtParser: JWTParser
+) {
+    fun isTokenValidAndNotBlank(token: String?): Boolean =
+        !token.isNullOrBlank() && isTokenValid(token)
```

Comment:
not sure how this eventually would work. Do the refresh token and accesstoken use the same validatetoken method?

Reply:
Yes, both are JWT and validated against a Keycloak Key